### PR TITLE
Docs maintenance: remove external images, article QA + updates

### PR DESCRIPTION
**Pull request by paulOsinski**
This PR is intended to move externally hosted images into the main repo, along with ongoing docs updates:

File Changes
- replace all externally hosted images with images on the repo, and update all image refs to use those hosted images

Article Changes
- Add Universal Importer / Dojo-CLI docs
- update DefectDojo Pro Changelog (these changes have already been implemented in Master via [this commit](https://github.com/paulOsinski/django-DefectDojo/commit/032960d57bd1185ac4bc4623ab65fa75ab348b22))
- rework Request A Trial article with updated language
- QA + update articles in Cloud Manager directory
- removed all support.defectdojo.com external links

[sc-8918]
[sc-9103]

### CODE REVIEW

File: docs/content/en/cloud_management/connectivity-troubleshooting.md [hunk 1]
```diff
@@ -5,69 +5,49 @@ description: "Reconnect to your DefectDojo Instance"
...
 
-
-
 Please note that this login method only works for the **Cloud Manager**, an admin site which your team members may not all have access to. Directly logging into your instance to use DefectDojo is only possible by directly connecting to **yourcompanyinstance.cloud.defectdojo.com/login**.
 
 
 
-# **I've lost access to my MFA codes**
-
+## **I've lost access to my MFA codes**
```
Comment (cneill) [inline docs/content/en/cloud_management/connectivity-troubleshooting.md:50]: Just a small suggestion to match the other headings above.

```suggestion
## I've lost access to my MFA codes
```

File: docs/content/en/cloud_management/using-cloud-manager.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Resources
+<https://cloud.defectdojo.com/resources/>
+
+The Resources page contains a Contact Us form, which you can use to get in touch with our Support team.
+
+![image](images/Using_the_Cloud_Manager_3.png)
+
+## Tools
+<https://staging.defectdojo.com/external_tools/defectdojo-cli>
```
Comment (cneill) [inline docs/content/en/cloud_management/using-cloud-manager.md:43]:
```suggestion
<https://cloud.defectdojo.com/external_tools/defectdojo-cli>
```

File: docs/content/en/connecting_your_tools/parsers/generic_findings_import.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- Url: Url associated with the finding.
+- Severity: Severity of the finding. Must be one of Info, Low, Medium, High, or Critical.
+- Description: Description of the finding. Can be multiple lines if enclosed in double quotes.
+- Mitigation: Possible Mitigations for the finding. Can be multiple lines if enclosed in double quotes.
+- Impact: Detailed impact of the finding. Can be multiple lines if enclosed in double quotes.
+- References: References associated with the finding. Can be multiple lines if enclosed in double quotes.
+- Active: Indicator if the finding is active. Must be empty, TRUE or FALSE
+- Verified: Indicator if the finding has been verified. Must be empty, TRUE, or FALSE
+- FalsePositive: Indicator if the finding is a false positive. Must be TRUE, or FALSE.
+- Duplicate:Indicator if the finding is a duplicate. Must be TRUE, or FALSE
```
Comment (cneill) [inline docs/content/en/connecting_your_tools/parsers/generic_findings_import.md:23]:
```suggestion
- Duplicate: Indicator if the finding is a duplicate. Must be TRUE, or FALSE
```

**Review by cneill — APPROVED** [review]

**Comment by paulOsinski** [comment]
This wraps up my docs maintenance work for the week - this can be merged as is and I will start a new PR for next week's work.

**Review by mtesauro — APPROVED** [review]
Approved

**Review by Maffooch — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**